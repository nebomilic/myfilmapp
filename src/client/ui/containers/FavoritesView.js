import React from 'react';
const FavoritesView = () => { 
    return (
        <div>
           This is favorites view!
        </div>  

    ); 
};   
export default FavoritesView;