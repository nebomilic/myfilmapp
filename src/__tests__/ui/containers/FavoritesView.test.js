import React from 'react';
import FavoritesView from '../../../client/ui/containers/FavoritesView';
import renderer from 'react-test-renderer';

it ('renders corectly', () => {
    const tree = renderer
        .create(<FavoritesView/>)
        .toJSON();

    expect(tree).toMatchSnapshot();
})