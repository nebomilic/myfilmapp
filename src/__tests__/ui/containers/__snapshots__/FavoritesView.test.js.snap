// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders corectly 1`] = `
<div>
  This is favorites view!
</div>
`;
